Fix export log failing when there is not enough logs

shjortConcordium opened the pull request:
## Purpose

Fix #288

## Changes

- Check if extra log file exists instead of assuming.
- Fix warnings from importing JSON incorrectly.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

orhoj commented:
I can now export the logs, but the format of the logs are not in JSON as expected. They seem to be Javascript-like objects.

orhoj reviewed (changes requested):
Fix the log format.

shjortConcordium commented:
> Fix the log format.

I have fixed the format so that it is JSON, and ensured that error messages are put into the logs. I have removed the stack trace, but they were useless in production anyway.

orhoj commented:
The JSON part seems fixed, but now the custom log message I provide seems to end up mixing with the provided error which is a little weird. Is there a way to ensure that the message we provide and the error are logged in separate fields? It will make querying the log file a lot easier. Right now it is:
```
{
   "message":"Error: Failed to connect before the deadline Error while checking for closing baker pools",
   "level":"error",
   "timestamp":"2022-05-06T05:36:10.404Z"
}
```
but preferably it should be:
```
{
   "message":"Error while checking for closing baker pools",
   "error":"Error: Failed to connect before the deadline",
   "level":"error",
   "timestamp":"2022-05-06T05:36:10.404Z"
}
```

orhoj commented on app/features/ChainDataSlice.ts:
```diff
@@ -88,7 +88,7 @@ export async function init(dispatch: Dispatch) {
         dispatch(setConsensusStatus(cs));
     } catch (e) {
-        window.log.error('Could initialize chain data state:', e);
+        window.log.error(e, 'Could initialize chain data state:');
```
```suggestion
        window.log.error(e, 'Could not initialize chain data state');
```

orhoj commented on app/pages/AccountCreation/GeneratePage/GeneratePage.tsx:
```diff
@@ -180,9 +180,7 @@ export default function AccountCreationGenerate({
-                    window.log.error(e, {
-                        message: ' caused creation of Account to fail.',
-                    });
+                    window.log.error(e, ' caused creation of Account to fail.');
```
Fix the log message here.

orhoj commented on app/pages/IdentityIssuance/ExternalIssuance/ExternalIssuance.tsx:
```diff
@@ -163,9 +163,7 @@ async function generateIdentity(
-        window.log.error(e, {
-            message: ' caused confirmation of Identity to fail.',
-        });
+        window.log.error(e, ' caused confirmation of Identity to fail.');
```
Fix the log message here.
